components/stacks-network/src/chainhooks.rs

```diff
@@ -1,14 +1,11 @@
+use chainhook_sdk::chainhook_types::{BitcoinNetwork, StacksNetwork};
```

Comment:
This `clarinet-cli/src/chainhooks/mod.rs` was only used in `stacks-network`, so I moved it

Reply:
When I first made this change, it allowed for some improved code reusability, but I ended up not needing it after all.

So, if anyone doesn't like this being moved I can put it back.